examples/apps/branding/app.py

```diff
@@ -0,0 +1,304 @@
+            "https://cdn.britannica.com/94/192794-050-3F3F3DDD/panels-electricity-order-sunlight.jpg",
+            margin=0,
+            sizing_mode="stretch_both",
+        ),
+        "# Reports\n\nDownload our annual report and see all our reports and presentations",
+        width=300,
+        sizing_mode="fixed",
+        margin=10,
+    ),
+    sizing_mode="fixed",
+    align="center",
+    margin=25,
+)
+
+
+@pn.cache
+def get_data(n_categories: int, n_rows=100):
+    x = np.random.rand(n_rows)
+    y = np.random.rand(n_rows)
+
+    categories = [f"Category {i+1}" for i in range(n_categories)]
+    category = np.random.choice(categories, n_rows)
+
+    dataframe = pd.DataFrame({"x": x, "y": y, "category": category}).sort_values(
+        "category"
+    )
+
+    return dataframe
+
+
+def get_categorical_plot(dark_theme: bool = False):
+    df = get_data(n_categories=3, n_rows=30)
+    cmap = colors.get_categorical_palette(n_colors=3, dark_theme=dark_theme)
+    return df.hvplot.scatter(
+        x="x",
+        y="y",
+        size=75,
+        color="category",
+        cmap=cmap,
+        height=350,
+        responsive=True,
+        title="Categorical Plot",
+        tools=["fullscreen"],
+        legend="top_right",
+    ).opts(backend_opts={"plot.toolbar.autohide": True}, toolbar="above")
+
+
+def get_continous_plot(dark_theme):
+    np.random.seed(42)
+    data = pd.DataFrame(
+        {
+            "x": np.random.rand(100),
+            "y": np.random.rand(100),
+            "value": np.random.rand(100) * 100,
+        }
+    )
+    cmap = colors.get_continous_cmap(dark_theme)
+    return data.hvplot.scatter(
+        x="x",
+        y="y",
+        c="value",
+        size=75,
+        cmap=cmap,
+        colorbar=True,
+        height=350,
+        responsive=True,
+        title="Continuous Plot",
+        tools=["fullscreen"],
+    ).opts(backend_opts={"plot.toolbar.autohide": True}, toolbar="above")
+
+
+categorical_plot = pn.bind(get_categorical_plot, dark_theme=state.param.dark_theme)
+continous_plot = pn.bind(get_continous_plot, dark_theme=state.param.dark_theme)
+
+df = pd.DataFrame(
+    {
+        "int": [1, 2, 3],
+        "float": [3.14, 6.28, 9.42],
+        "str": ["A", "B", "C"],
+        "bool": [True, False, True],
+        "date": [dt.date(2019, 1, 1), dt.date(2020, 1, 1), dt.date(2020, 1, 10)],
+        "datetime": [
+            dt.datetime(2019, 1, 1, 10),
+            dt.datetime(2020, 1, 1, 12),
+            dt.datetime(2020, 1, 10, 13),
+        ],
+    },
+    index=[1, 2, 3],
+)
+tabulator_formatters = {
+    "float": {
+        "type": "progress",
+        "max": 10,
+        "color": [
+            colors.LIGHT_THEME.success,
+            colors.LIGHT_THEME.warning,
+            colors.LIGHT_THEME.error,
+        ],
+    },
+    "bool": {
+        "type": "tickCross",
+        "tickElement": f'<span class="material-icons-outlined" style="color: {colors.LIGHT_THEME.error}">check</span>',
```

Comment:
I can't get custom material (or bootstrap) icons working in the Tabulator table.

![image](https://github.com/user-attachments/assets/25359b09-ab1d-4eef-8fd8-39a1bb0f9b4c)